src/pages/DatasetRegistration.js

```diff
@@ -702,44 +702,33 @@ class DatasetRegistration extends Component {
             a({
               id: 'btn_step_1',
               onClick: this.step1,
-              className: 'col-lg-3 col-md-3 col-sm-12 col-xs-12 access-color jumbotron box-vote multi-step-title '
+              className: 'col-lg-3 col-md-3 col-sm-12 col-xs-12 dataset-color jumbotron box-vote multi-step-title '
                 + (this.state.step === 1 ? 'active' : '')
             }, [
               small({}, ['Step 1']),
-              'Researcher Information',
+              'Dataset Information',
```

Comment:
Thank you for casing `Dataset` correctly - it is cased incorrectly in many, many places.